lacommunaute/forum_conversation/tests/tests_views.py

```diff
@@ -707,6 +708,23 @@ def test_queryset_filtered_on_tag(client, db, tag):
     assertNotContains(response, other_topic.subject)
 
 
+def test_queryset_for_tagged_topic(client, db, snapshot):
+    tags = ["buckley", "jeff"]
+    tagged_topic = TopicFactory(with_post=True, with_tags=tags)
+    untagged_topic = TopicFactory(with_post=True)
+
+    response = client.get(reverse("forum_conversation_extension:topics") + f"?tags={tags[0]}")
+    content = parse_response_to_soup(response, selector="#topic-list-filter-header")
+    assert str(content) == snapshot(name="tagged_topic")
+    assertContains(response, tagged_topic.subject, status_code=200)
```

Comment:
C’est la valeur par défaut.
```suggestion
    assertContains(response, tagged_topic.subject)
```